src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py

```diff
@@ -80,91 +78,33 @@ def create_figure(self) -> Figure:
         color_selector = ColorSelectorFactory.create_instance(coloring_rule='unique')
         if self._case == 'best':
             color_selector.get_color()
-        stacked_bar_source = StackedBarSource(target_objects)
-        fig.add_tools(stacked_bar_source.create_hover())
-        stacked_bar_data, x_width_list = \
-            self._get_stacked_bar_data(data, y_labels, self._xaxis_type, x_label)
-
-        vbar_stacks = []
-        for y_label in y_labels:
-            vbar_stack = fig.vbar_stack(
-                [y_label],
-                x='start time',
-                width='x_width_list',
-                source=stacked_bar_source.generate(y_label, stacked_bar_data, x_width_list),
-                color=color_selector.get_color(y_label)
-            )
-            vbar_stacks.append((y_label, vbar_stack))
-
-        legend = Legend(items=vbar_stacks, location='bottom_left')
+        colors = [color_selector.get_color(y_label) for y_label in y_labels]
+
+        source = StackedBarSource(data, y_labels, self._xaxis_type, x_label)
+        # stack bottom -> top to top -> bottom
```

Comment:
This comment needs a little more clarification.

Reply:
Fixed it in [dbdc031](https://github.com/tier4/CARET_analyze/pull/307/commits/dbdc0311e5097a031aa934d83f464cacba3d933b).